source/fe/fe_data.cc

```diff
@@ -68,8 +73,10 @@ FiniteElementData<dim>::operator==(const FiniteElementData<dim> &f) const
   return ((dofs_per_vertex == f.dofs_per_vertex) &&
           (dofs_per_line == f.dofs_per_line) &&
           (dofs_per_quad == f.dofs_per_quad) &&
-          (dofs_per_hex == f.dofs_per_hex) && (components == f.components) &&
-          (degree == f.degree) && (conforming_space == f.conforming_space));
+          (dofs_per_hex == f.dofs_per_hex) &&
+          (non_local_dofs_per_cell == f.non_local_dofs_per_cell) &&
+          (components == f.components) && (degree == f.degree) &&
+          (conforming_space == f.conforming_space));
```

Comment:
nice catch -- these last two lines are uncontroversial and independent of the current discussion. Would you mind putting them into a separate PR?

Reply:
but those were already there, it's just indentation, AFAICT. I guess to make the changelog smaller, one could add the new `&&` at the very end.

Reply:
Oh, I missed that. I'm going to state (again :-( ) that the new automatic indentation has diminished readability :-(